cxplat/inc/cxplat.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+#pragma once
+
+#include "cxplat_fault_injection.h"
+#include "cxplat_memory.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    cxplat_status_t
+    cxplat_initialize();
```

Comment:
Would you mind not having the spaces before all these lines? IMHO, it's a lot of wasted space.

Reply:
I think clang-format did it, the formatting isn't supposed to be manual.
We can see if there's a config option to do what you want.